WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostRepository.kt

```diff
@@ -120,22 +121,26 @@ class EditPostRepository
         this.post = postForUndo?.clone()
     }
 
-    fun saveSnapshot() {
-        postSnapshotWhenEditorOpened = post?.clone()
+    fun postHasChangesFromDb(post: PostModel): Boolean =
+            postSnapshotForDb == null || post != postSnapshotForDb
+
+    fun saveDbSnapshot(post: PostModel) {
+        postSnapshotForDb = post.clone()
     }
 
-    fun isSnapshotDifferent(): Boolean =
-            postSnapshotWhenEditorOpened == null || post != postSnapshotWhenEditorOpened
+    fun saveInitialSnapshot() {
+        postSnapshotWhenEditorOpened = post?.clone()
+    }
 
-    fun hasSnapshot() = postSnapshotWhenEditorOpened != null
+    fun hasInitialSnapshot() = postSnapshotWhenEditorOpened != null
```

Comment:
We might want to consider either keeping the name `SnapshotWhenEditorOpened` even for the method or renaming the field to `initialSnapshot`. Using two different terms for the same thing is a bit confusing. Wdyt?

Reply:
Renamed in https://github.com/wordpress-mobile/WordPress-Android/pull/10797/commits/2e10f7a8d5a78efbe3ff6e724bfbb0be531c8302

Reply:
I found one more [method](https://github.com/wordpress-mobile/WordPress-Android/blob/2e10f7a8d5a78efbe3ff6e724bfbb0be531c8302/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostRepository.kt#L143) which we probably want to rename as well. Wdyt?

Reply:
updated here https://github.com/wordpress-mobile/WordPress-Android/pull/10797/commits/e85fa23820484e5fa69831d8379a41d22cd9c59c